### ISSUE TITLE
I appreciate this add on however, I am noticing that often I am missing one student from the attendance that was most definitely in the meeting.  Is there something in settings I need to change?

### ISSUE THREAD

**Issue by KBauldree** [al-caughey/Google-Meet-Attendance#61]

**Comment by AlCaughey** [comment]
I have to believe there is a bug but I don't know what or why ATM...

**Comment by Eagle-Man** [comment]
@KBauldree Do you have your students entered in a sorted class list?  Where would this missing student be on the list?

**Comment by KBauldree** [comment]
> @KBauldree Do you have your students entered in a sorted class list? Where would this missing student be on the list?

For example I have 21 participants that come to the meeting.  However, when the attendance is generated it is only showing 20.

**Comment by Eagle-Man** [comment]
I understand that part, but I'm trying to get more information which might lead to finding the solution.  Are the names in your class list sorted?  If so, is it the first or last person that is missing?  Someone random?  Is it the first/last person to arrive in the Meet?  Do they join while you are presenting?

**Comment by KBauldree** [comment]
Okay, so I would say it is random.  However, many times it has been the first person.  Sometimes students are waiting for me but sometimes they join after.  When I say they are waiting, as soon as I open the meeting they pop in pretty quick.  So I am assuming they he the join button before I opened it.

**Comment by Eagle-Man** [comment]
Ah.  So do you think it is one of those early people from before you manage to get in, or do you think it’s the first person after you get in?

@Al: could it be that there’s a bug when a large group enters at once?

**Comment by KBauldree** [comment]
No not always.  It could be but I don't think that is always the case as to who is missing.  Other staff are using the same add on and not experiencing the same issue.

**Comment by AlCaughey** [comment]
When a larger group joins at once, Google shows `Eric F and 4 other
joined`  I currently pick off the `Eric F` and the rely on Grid View to
catch the others (because Google does not provide a list of who the 4 other
are)

I've been checking here and still cannot replicate the issue

**Comment by Eagle-Man** [comment]
One more thought... @KBauldree, is the "Show only participants with video" option in Grid View turned off?

**Comment by KBauldree** [comment]
No, it on.

**Comment by Eagle-Man** [comment]
Aha!  That would be the cause.  The one who is not in attendance does not have their video on, so Grid View has removed them from view. Therefore, Attendance cannot see them either.  If you turn off that setting, they will appear as a blank box with their name, and Attendance will be able to record their presence.

**Comment by KBauldree** [comment]
We require all of our students to have on their cameras.

**Comment by Eagle-Man** [comment]
Okay, but ARE they all on?  This seems to indicate no.  If the school mandates cameras on, it’s best to turn off that setting anyway because then you can see which cameras are off.

**Comment by KBauldree** [comment]
I can turn of the setting but I can guarantee that some of the kids that have been missing from the attendance have had their cameras on.

**Comment by ailiou69** [comment]
I'm having a similar issue.  Several students were on my Meet but didn't show up on the auto saved attendance report.  I was able to save the attendance in excel which is another reason I know there is a discrepancy  Students were uploaded to an excel spreadsheet with their times on and off.  It was different from the auto download.  So i will just save to excel during class to make sure i get everyone.

**Comment by Eagle-Man** [comment]
@ailiou69 Do you mean you download the CSV file manually?  Are you using Grid View (fix) or Meet's new extended Tiled view?

@ailiou69 , @KBauldree - Are these students not getting check marks in the attendance window in Meet, or is this just an issue with the HTML report?  Also, do you have pre-set class lists for attendance to check against, or do you just let the list populate automatically?

**Comment by ailiou69** [comment]
Yes, and yes. Some students are not getting check marks.  I'm not sure what
you mean by the last part.  I think it just populates automatically as
students sign on.  Can I create a class list for the extension to use?

**Comment by Eagle-Man** [comment]
@ailiou69 Which are you using, Grid View (fix) or Meet?

What I mean is that you can create class lists of the expected attendees, and then it will check them off as they arrive.  Then any without a check mark were absent; any who arrive who are not on the list will be added to the list with a ?.  Check out the Getting Started video Al made, or see the help pages in the extension.  The other way is to clear the default list each time and let it auto-populate (you'll have to find absent people manually).  It's fine to do it either way, but I'm just trying to narrow it down.  Sounds like you use the automatic method.  It's possible it's an issue with the auto-add, but there's not enough evidence for that yet to say for sure.

**Comment by KBauldree** [comment]
I use grid view fix.  This week I have made sure to open my meeting early enough so that no one is waiting.  Seems to fix the problem.

Karri Bauldree
Senior Health
JH PE/Health
Salem Academy
________________________________

**Comment by Eagle-Man** [comment]
@KBauldree From what I hear, it's not a constant/consistent problem.  Do you have a class list that the extension is checking or do you let the extension generate the list as people enter?

**Comment by KBauldree** [comment]
I let it generate

Karri Bauldree
Senior Health
JH PE/Health
Salem Academy
________________________________

**Comment by woodssowild** [comment]
I think I have noticed that students who arrive before I do will not show up on the attendance.
